src/sidebar/components/hooks/use-element-should-close.js

```diff
@@ -42,7 +42,11 @@ function useElementShouldClose(closeableEl, isOpen, handleClose) {
       document.body,
       'focus',
       event => {
-        if (!closeableEl.current.contains(event.target)) {
+        // preact element or just a dom element?
+        const node = closeableEl.current.base
```

Comment:
This was not working out of the box on a top level preact component. From my research the "base" object is a solid test to check

Reply:
I'm not sure if the `base` property of Preact components is considered part of its public API. I've asked on their Slack channel for confirmation.

In any case, as long as you're passing a ref to a DOM element in, I don't think this should be necessary. If you do think it is, can you clarify why?